zowe/zowe-cli-scs-plugin
# chore: use Secrets SDK instead of keytar

**What It Does**

This PR replaces keytar with the Secrets SDK `keyring` module.

**How to Test**

Install v1 of the CLI from [this PR](https://github.com/zowe/zowe-cli/pull/1804), and then install the SCS plugin from this branch (`npm run installPlugin`). Reviewers should be able to execute a CLI command, e.g. `zowe files list ds ...` with a v1 profile that's already set up, and the CLI should be able to access the secured credentials.

**Review Checklist**
I certify that I have:
